docusaurus.config.js

```diff
@@ -0,0 +1,203 @@
+// @ts-check
+
+//#region Types
+/**
+ * @typedef DocusaurusSearchLocalOptions
+ * @property {boolean} [indexDocs=true] Whether to index docs.
+ * @property {boolean} [indexBlog=true] Whether to index blog.
+ * @property {boolean} [indexPages=false] Whether to index pages.
+ * @property {string|string[]} [docsRouteBasePath='/docs'] Base route path(s) of docs. Slash at beginning is not required.
+ * @property {string|string[]} [blogRouteBasePath='/blog'] Base route path(s) of blog. Slash at beginning is not required.
+ * @property {string|string[]} [language='en'] All [lunr-languages](https://github.com/MihaiValentin/lunr-languages) supported languages.
+ * @property {boolean} [hashed=false] Whether to add a hashed query when fetching index (based on the content hash of all indexed `*.md` in {@link DocusaurusSearchLocalOptions.docsDir `docsDir`} and {@link DocusaurusSearchLocalOptions.blogDir `blogDir`} if applicable).
+ * @property {string|string[]} [docsDir='docs'] The dir(s) of docs to get the content hash, it's relative to the dir of your project.
+ * @property {string|string[]} [blogDir='blog'] Just like the {@link DocusaurusSearchLocalOptions.docsDir `docsDir`} but applied to blog.
+ * @property {boolean} [removeDefaultStopWordFilter=false] Sometimes people (E.g., us) want to keep the English stop words as indexed, since they maybe are relevant in programming docs.
+ * @property {boolean} [highlightSearchTermsOnTargetPage=false] Highlight search terms on target page.
+ * @property {number} [searchResultLimits=8] Limit the search results.
+ * @property {number} [searchResultContextMaxLength=50] Set the max length of characters of each search result to show.
+ * @property {{[key: string]: string}} [translations] Set translations of this theme.
+ * @property {string|RegExp|(string | RegExp)[]} [ignoreFiles] Set the match rules to ignore some files.
+ */
+//#endregion
+
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
+const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
+const creationYear = 2016;
+const url = dev ? process.env.DEV_URL : process.env.PROD_URL;
+// Note to self, urls should'nt be finished with a slash (/)
+const url = dev
```

Comment:
What's this line doing?

Reply:
Nothing, it's just me that messed up with git 😅.
That's the cause too of why `dart_logo.svg` is broken

Reply:
Ok, so, it seems that github don't accept external URLs in svgs.
But it works locally:
![image](https://user-images.githubusercontent.com/74512338/165082270-c131545f-2755-43f5-a273-57ee66fcc316.png)

Reply:
whats the point in using image tag inside svg? Cant you just use it directly?

Reply:
I can't use it directly because of the CORS origin policy which disallows importing the svg directly from the URL.

Reply:
there is no raster graphic logo available to use then?

Reply:
Well, we can.. but that would break the consistency of `FeatureItem`